server/src/models/User.js

```diff
@@ -22,8 +22,15 @@ const UserSchema = new Schema({
     minLength: 2,
     maxLength: 50,
   },
-  resume_url: {
-    type: String,
+  resume: {
+    url: {
+      type: String,
+      required: true,
+    },
+    text: {
```

Comment:
Why are we storing both the url to the resume and the parsed PDF?

Reply:
should I separate it? cause I thought it should be bundled into 1 object

Reply:
My understanding was that the frontend only display the url to the PDF on s3. Is that not the case?

Reply:
oh sure... But I wonder where should I leave the parsed text? should I create separate model for this?